lib/Controller/ApiController.php

```diff
@@ -129,11 +129,11 @@ private function checkAuthorized() {
 			return false;
 		}
 
-		if (!$this->groupManager->isAdmin($user->getUID())) {
-			return false;
+		if ($this->groupManager->isAdmin($user->getUID())) {
+			return true;
 		};
 
-		return true;
+		return false;
```

Comment:
What about?

``return $this->groupManager->isAdmin($user->getUID());``

It returns the desired boolean already.

Reply:
Make sense, but I'd prefer to merge https://github.com/nextcloud/serverinfo/pull/288/files#diff-53e5a55431f07eddf1b35d8e80157e02bb1955bda3e3fcaf20bb6dc8c1f88e56 first then.

Reply:
thanks! reviewed it, also needs a rebase.